leafletmap/src/test/resources/de/saring/leafletmapdemo/demo.fxml

```diff
@@ -8,18 +8,19 @@
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox spacing="16.0" xmlns="http://javafx.com/javafx/8.0.65" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.saring.leafletmapdemo.Controller">
+<VBox spacing="16.0" xmlns="http://javafx.com/javafx/8.0.141" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.saring.leafletmapdemo.Controller">
    <children>
       <StackPane fx:id="spMapView" prefHeight="480.0" prefWidth="640.0" VBox.vgrow="ALWAYS" />
       <HBox alignment="CENTER_LEFT" spacing="16.0">
          <children>
             <Button fx:id="btDisplayTrack" mnemonicParsing="false" onAction="#onDisplayTrack" text="Display Track" />
+            <Button fx:id="btClear" mnemonicParsing="false" onAction="#onClear" text="Clear" />
             <Label text="Position:">
                <HBox.margin>
                   <Insets left="8.0" />
                </HBox.margin>
             </Label>
-            <Slider fx:id="slPosition" max="0.0" HBox.hgrow="ALWAYS" />
+            <Slider fx:id="slPosition" max="0.0" prefHeight="14.0" prefWidth="300.0" HBox.hgrow="ALWAYS" />
```

Comment:
Changes are not needed, please revert them.

Reply:
That seems to be an artifact from the fxml editor in netbeans - I didn't put anything there.

Reply:
Done